wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/devfile/server/DevfileModule.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.workspace.devfile.server;
```

Comment:
Is this copied from the previous `DevfileModule` or different file? If the former, it'd be great to keep the git history with a move, but it's altogether not that important.

Reply:
It was definitely moved. Not sure why git considers it remove and add.

Reply:
Yeah sometimes it breaks for me (especially if the move is done via IDE).